### PR TITLE
Update code generation to GridTools v1.0.1

### PR DESCRIPTION
**Pull request by cosunae**

### CODE REVIEW

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:cosunae/dawn.git"
+    GIT_TAG "kesch_gcc5.4"
```
Comment (cosunae) [inline bundle/CMakeLists.txt]:
```suggestion
    GIT_TAG "update_gt_v1.0.1" 
```

**Comment by cosunae** [comment]
launch jenkins

**Comment by havogt** [comment]
launch pipeline

File: test/integration-test/CodeGen/asymmetric_benchmark.cpp [hunk 2]
```diff
@@ -55,8 +55,8 @@ TEST(asymmetric, test) {
   dawn_generated::OPTBACKEND::asymmetric_stencil stencil_opt(dom, in, out_opt);
```
Comment (havogt) [inline test/integration-test/CodeGen/asymmetric_benchmark.cpp]: It's passed twice now? To construction and to run()?

**Comment by havogt** [comment]
launch pipeline

**Comment by havogt** [comment]
launch pipeline

**Comment by havogt** [comment]
launch pipeline

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch jenkins

**Pull request merged**